R/pkg/R/DataFrame.R

```diff
@@ -455,6 +455,17 @@ setMethod("createOrReplaceTempView",
               invisible(callJMethod(x@sdf, "createOrReplaceTempView", viewName))
           })
 
+#' @param tableName A character vector containing the name of the table
+#' @rdname createOrReplaceTempView
+#' @name registerTempTable
```

Comment:
on that note the docs might look nice if the params had the same name (hmm if we use viewName as the param it might break somebody using named arguments, but not sure if thats common ?)